src/main/java/io/jenkins/plugins/restlistparam/RestListParameterDefinition.java

```diff
@@ -251,15 +275,15 @@ public Integer getDefaultCacheTime() {
     }
 
     @POST
-    public FormValidation doCheckRestEndpoint(@AncestorInPath final Item context,
+    public FormValidation doCheckRestEndpoint(@AncestorInPath final hudson.model.Item context,
```

Comment:
Exactly for this reason I am not a big fan of naming your model class `Item` :thinking: (however I am no big fan of the name for the Jenkins Core class either mind you)